.github/workflows/ci.yml

```diff
@@ -220,9 +220,9 @@ jobs:
           # TODO: would like to adopt `actionlint` pre-commit hook
           # but false positive here and inability to do an inline ignore
           # prevents this https://github.com/rhysd/actionlint/issues/237
-          - python-version: ${{ github.event_name != 'schedule' && '3.9' }}
-          - python-version: ${{ github.event_name != 'schedule' && '3.10' }}
-          - python-version: ${{ github.event_name != 'schedule' && '3.11' }}
```

Comment:
whoohoo! so this will ensure that the full workflow will run on `schedule` and `push tag`

Reply:
Correct